safe-global/safe-wallet-web
# feat: zkEmail fake door [SW-564]

## What it solves

Resolves [SW-564](https://www.notion.so/safe-global/zkEmail-fake-door-1448180fe57380bc8e45c2e976afb426)

## How this PR fixes it

- adds a new recovery option for zkEmail
- proceeding with this new option selected will show a modal with some info as fake door

## How to test it

1. Open the security settings
2. Click on "Set up recovery"
3. Observe a new option "Email recovery" be present in the modal that opens
4. Select this new option and click the "Set up" button
5. Observe a new modal appear with some "coming soon" information

## Screenshots

![Screenshot 2024-11-28 at 17 50 07](https://github.com/user-attachments/assets/fd0f086e-7217-4c05-a04c-84d1a06c7d82)
![Screenshot 2024-11-28 at 17 50 25](https://github.com/user-attachments/assets/41077fdb-a002-497c-b47d-bbbe016f26db)
![Screenshot 2024-11-28 at 17 53 53](https://github.com/user-attachments/assets/fec74f8e-e230-4db9-82e8-5e5e421570a8)

## Checklist
